README.md

```diff
@@ -108,32 +108,89 @@ Note that you may need to remove any existing data in `results/` in order for sn
 
 
-#### 1. Data is stored on a private S3 bucket
+### Use additional metadata and/or sequences
 
-The phylogenetics pipeline starts by downloading data from a private S3 bucket.
-If you don't have credentials for this bucket you can run the build using the example data provided in this repository.
-Before running the build, copy the example sequences and metadata into the `data/` directory like so:
+Using a config YAML provided via `--configfile`, or the same data via `--config` you can supply additional inputs.
+For instance, this repo has a small set of example metadata + HA sequences.
+We could add these to the default inputs via:
 
-```bash
-mkdir -p data/
-cp -r -v example_data/* data/
+```yaml
+additional_inputs:
+  - name: example-data
+    metadata: example_data/gisaid/metadata.tsv
+    sequences:
+        ha: example_data/gisaid/sequences_ha.fasta
+```
+
+Which will merge `example_data/gisaid/metadata.tsv` with the default metadata, and add the sequences from `example_data/gisaid/sequences_ha.fasta` to the default HA sequences (all other segments will just use the default sequences).
+
+If you had sequences for each segment you could use a wildcard for the segment like so:
+
+```yaml
+additional_inputs:
+  - name: example-data
+    metadata: example_data/gisaid/metadata.tsv
+    sequences: example_data/gisaid/sequences_{segment}.fasta
 ```
 
-Then run the build with the test target, a H5N1 HA "all time" tree:
+
+> NOTE: These added data will be subject to the same filtering rules as the starting data.
+  At a minimum, you'll want to ensure new sequences have metadata which defines their subtype and date, as filter steps will prune out those without valid values here.
+
+> NOTE: Metadata merging is via `augur merge` and we add one-hot columns for each input source with the column name `input_{NAME}`, for instance the above example would have a `input_secret` column with values of `1` for metadata rows which were included in `secret.tsv` and `0` otherwise.
+  You can use this for additional filtering commands as needed.
+
+
+### Replace the starting metadata and sequences
+
+Using the same approach as above but replacing `additional_inputs` with `inputs` will replace the default inputs.
```

Comment:
I was initially confused why the config needed both `inputs` and `additional_inputs` until I saw this and remembered Snakemake doesn't merge lists.

Thoughts on changing `inputs` to a dict where the key is the name so the `inputs` configs get merged?

```yaml
inputs: 
  local-ingest:
    metadata: ingest/fauna/results/metadata.tsv
    sequences: ingest/fauna/results/sequences_{segment}.fasta
  example-data:
    metadata: example_data/gisaid/metadata.tsv
    sequences:
      ha: example_data/gisaid/sequences_ha.fasta
```

Reply:
Thanks for reviewing! Much appreciated

> Thoughts on changing inputs to a dict where the key is the name so the inputs configs get merged?

I deliberately chose lists because of their merging behaviour. Using a list for `inputs` allows us to trivially replace the default data (see "Using locally ingested data" in README) or drop it entirely. Using a second list for `additional_inputs` allows us to easily add additional data sources, and the explicitness of the name is nice when seen in the context of config overlays.